gravatar-ui/api/gravatar-ui.api

```diff
@@ -146,34 +151,45 @@ public final class com/gravatar/ui/components/ComposableSingletons$ProfileSummar
 }
 
 public final class com/gravatar/ui/components/ProfileSummaryKt {
 	public static final fun ProfileSummary (Lcom/gravatar/api/models/Profile;Landroidx/compose/ui/Modifier;Landroidx/compose/runtime/Composer;II)V
-	public static final fun ProfileSummary (Lcom/gravatar/ui/components/ComponentState;Landroidx/compose/ui/Modifier;Lkotlin/jvm/functions/Function3;Lkotlin/jvm/functions/Function3;Landroidx/compose/runtime/Composer;II)V
+	public static final fun ProfileSummary (Lcom/gravatar/restapi/models/Profile;Landroidx/compose/ui/Modifier;Landroidx/compose/runtime/Composer;II)V
+	public static final fun ProfileSummary (Lcom/gravatar/ui/components/ComponentState;Landroidx/compose/ui/Modifier;Landroidx/compose/runtime/Composer;II)V
+	public static final fun ProfileSummaryWithComponentState (Lcom/gravatar/ui/components/ComponentState;Landroidx/compose/ui/Modifier;Lkotlin/jvm/functions/Function3;Lkotlin/jvm/functions/Function3;Landroidx/compose/runtime/Composer;II)V
 }
 
 public final class com/gravatar/ui/components/atomic/AboutMeKt {
 	public static final fun AboutMe (Lcom/gravatar/api/models/Profile;Landroidx/compose/ui/Modifier;Landroidx/compose/ui/text/TextStyle;Lkotlin/jvm/functions/Function4;Landroidx/compose/runtime/Composer;II)V
-	public static final fun AboutMe (Lcom/gravatar/ui/components/ComponentState;Landroidx/compose/ui/Modifier;Landroidx/compose/ui/Modifier;Landroidx/compose/ui/text/TextStyle;Lkotlin/jvm/functions/Function4;Landroidx/compose/runtime/Composer;II)V
+	public static final fun AboutMe (Lcom/gravatar/restapi/models/Profile;Landroidx/compose/ui/Modifier;Landroidx/compose/ui/text/TextStyle;Lkotlin/jvm/functions/Function4;Landroidx/compose/runtime/Composer;II)V
 	public static final fun AboutMe (Lcom/gravatar/ui/components/ComponentState;Landroidx/compose/ui/Modifier;Landroidx/compose/ui/text/TextStyle;Lkotlin/jvm/functions/Function4;Landroidx/compose/runtime/Composer;II)V
+	public static final fun AboutMeWithComponentState (Lcom/gravatar/ui/components/ComponentState;Landroidx/compose/ui/Modifier;Landroidx/compose/ui/Modifier;Landroidx/compose/ui/text/TextStyle;Lkotlin/jvm/functions/Function4;Landroidx/compose/runtime/Composer;II)V
 }
 
 public final class com/gravatar/ui/components/atomic/AvatarKt {
 	public static final fun Avatar-DzVHIIc (Lcom/gravatar/api/models/Profile;FLandroidx/compose/ui/Modifier;Lcom/gravatar/AvatarQueryOptions;Landroidx/compose/runtime/Composer;II)V
 	public static final fun Avatar-DzVHIIc (Lcom/gravatar/ui/components/ComponentState;FLandroidx/compose/ui/Modifier;Lcom/gravatar/AvatarQueryOptions;Landroidx/compose/runtime/Composer;II)V
+	public static final fun AvatarWithComponentState (Lcom/gravatar/restapi/models/Profile;FLandroidx/compose/ui/Modifier;Lcom/gravatar/AvatarQueryOptions;Landroidx/compose/runtime/Composer;II)V
```

Comment:
I think this one should have the Jvm signature modified, no? It doesn't take a generic object as a param.